src/app/directives/raml-initializer.js

```diff
@@ -9,30 +9,33 @@
       controller: ['$scope', '$window', function($scope, $window) {
         $scope.ramlUrl    = '';
 
-        ramlParserWrapper.onParseError(function(error) {
-          /*jshint camelcase: false */
-          var context = error.context_mark || error.problem_mark;
-          /*jshint camelcase: true */
+        ramlParserWrapper.onParseError(function(errors) {
```

Comment:
Sorry I was commenting on the `dist`, but the comments are still valid.

`theError` I would expect that name for a singleton.

This entire block is the same in `resources.js`